pkg/ansibleapp/types.go

```diff
@@ -0,0 +1,28 @@
+package ansibleapp
+
+import (
+	"encoding/json"
+)
+
+type Spec struct {
+	Id       string `json:"id"`
+	Name     string `json:"name"`
+	Bindable bool   `json:"bindable"`
+
+	// required, optional, unsupported
+	Async string `json:"async"`
+}
```

Comment:
+1 makes sense based one what I read in the api spec. The key bindable and async are important.